ipatests/prci_definitions/nightly_master_testing.yaml

```diff
@@ -480,6 +480,18 @@ jobs:
         timeout: 10800
         topology: *master_1repl
 
+  fedora-30/test_installation_TestInstallReplicaAgainstSpecificServer:
+    requires: [fedora-30/build]
+    priority: 50
+    job:
+      class: RunPytest
+      args:
+        build_url: '{fedora-30/build_url}'
```

Comment:
Need to add
```
        update_packages: True
```
for this nightly def (this nightly is doing a dnf update on all packages to install the packages available from updates-testing repo).